scripts/clouddata/syncSLErepos

```diff
@@ -152,9 +153,9 @@ for version in 4 5; do
     mount_and_rsync "/mnt/dist/ibs/Devel\:/Cloud\:/$version\:/Staging/images/iso/SUSE-CLOUD-$version-x86_64-Build[0-9][0-9][0-9][0-9]-Media1.iso" SUSE-Cloud-$version-devel-staging
 
     $rsync /mnt/euklid/mirror/SuSE/zypp-patches.suse.de/x86_64/update/SUSE-CLOUD/$version-POOL/ /srv/nfs/repos/SUSE-Cloud-$version-Pool/
-    $rsync /mnt/euklid/mirror/SuSE/build-ncc.suse.de/SUSE/Updates/SUSE-CLOUD/$version/x86_64/update/ /srv/nfs/repos/SUSE-Cloud-$version-Updates/
-    if test -d /mnt/dist/ibs/SUSE\:/Maintenance\:/Test\:/Cloud\:/$version\:/$arch/update/; then
-        $rsync /mnt/dist/ibs/SUSE\:/Maintenance\:/Test\:/Cloud\:/$version\:/$arch/update/ /srv/nfs/repos$repo_subdir/SUSE-Cloud-$version-Updates-test/
+    $rsync $buildncc/SUSE/Updates/SUSE-CLOUD/$version/x86_64/update/ /srv/nfs/repos/SUSE-Cloud-$version-Updates/
+    if test -d $ibsmaint/Cloud\:/$version\:/$arch/update/; then
+        $rsync $ibsmaint/Cloud\:/$version\:/$arch/update/ /srv/nfs/repos/$arch/SUSE-Cloud-$version-Updates-test/
```

Comment:
Looks like a bug in the old script, but there's no $arch here in this loop.

Which makes me wonder: do you want to also move the repos to the arch-specific subdirs for 4 and 5? (If yes, it's missing here)